### PR TITLE
Update defaultMode 0640 for all secret volume definitions: release-4.5

### PR DESCRIPTION
**Pull request by shalver**
Issue: https://github.ibm.com/alchemy-containers/armada-update/issues/2037

### CODE REVIEW

File: assets/kube-apiserver/kube-apiserver-deployment.yaml [hunk 1]
```diff
@@ -347,6 +349,7 @@ spec:
       - name: kms-kp
         secret:
           secretName: kp-wdek-secret
+          defaultMode: 0640
```
Comment (jmcmeek) [inline assets/kube-apiserver/kube-apiserver-deployment.yaml:352]: defaultMode was already set to `0440` 2 lines down.

Comment (shalver) [reply assets/kube-apiserver/kube-apiserver-deployment.yaml]: thx John, good catch; fixed up now

**Review by jmcmeek — APPROVED** [review]
lgtm

**Review by rtheis — APPROVED** [review]
/lgtm

**Pull request merged**